examples/tests/sdk_tests/queue/retention_period.main.w

```diff
@@ -1,14 +1,16 @@
 bring cloud;
 bring util;
 
-let var timeout = 100ms;
-let var retentionPeriod = 1s;
+let var timeout = 30s;
+let var retentionPeriod = 60s;
 
 let q = new cloud.Queue(timeout: timeout, retentionPeriod: retentionPeriod);
 
 test "retentionPeriod" {
   q.push("hello", "world");
 
+  util.sleep(retentionPeriod);
+
   assert(util.waitUntil(() => {
     return q.approxSize() == 0;
   }));
```

Comment:
Can you please increase the test timeout? It's 30s in aws (even though the default is 1m, which is still too close)
https://github.com/winglang/wing/blob/4248407b2f90fc19791da7159c4aff0072cf5edf/libs/wingsdk/src/target-tf-aws/function.ts#L207-L209
Have you tested it on the aws target?

Reply:
increased timeout and tested locally, good callout